src/modules/people-list/people-list-item.type.tsx

```diff
@@ -0,0 +1,44 @@
+export interface IProps {
+  /** clicked on card */
+  onUserClicked?: () => void;
+
+  /** user name */
+  name?: string | undefined;
+
+  /** user marital status */
+  maritalStatus?: string | undefined;
+
+  /** user gender */
+  gender?: string | undefined;
+
+  /** church entity name */
+  churchEntityName?: string | undefined;
+
+  /** profile pic url */
+  profilePic?: string | undefined | null;
+
+  /** user id */
+  userId?: string;
+
+  /** footer element */
+  footerElement?:
+    | React.ComponentType<any>
+    | React.ReactElement
+    | null
+    | undefined;
+
+  /** swipe left buttons */
+  leftButtons?: any | null;
+
+  /** swipe right button */
+  rightButtons?: any | null;
```

Comment:
I we are expecting JSX in `leftButtons` and `rightButtons` props then this should be `React.ReactElement | null | undefined` instead of `any | null`

Reply:
i have added React.ReactElement | null | undefine